File: jobs/2026-04-30__01-30-31/ch_python_cold_start_handler__eLq5HJU/artifacts/query.py
import os
import clickhouse_connect

host = os.environ["CH_HOST"]
port = int(os.environ["CH_PORT"])
user = os.environ["CH_USER"]
password = os.environ["CH_PASSWORD"]

client = clickhouse_connect.get_client(
    host=host,
    port=port,
    username=user,
    password=password,
    connect_timeout=30,
    send_receive_timeout=120,
)

result = client.query("SELECT 1")
print(result.result_rows)

File: jobs/2026-04-30__18-48-35/node_js_streaming_query_executio__cYd9SSF/artifacts/code/query.js
const { createClient } = require('@clickhouse/client');

// Get credentials from environment variables
const host = process.env.CLICKHOUSE_HOST;
const user = process.env.CLICKHOUSE_USER;
const password = process.env.CLICKHOUSE_PASSWORD;

// Create ClickHouse client with HTTPS connection on port 8443
const client = createClient({
  host: `https://${host}:8443`,
  username: user,
  password: password,
});

async function queryTables() {
  try {
    // Query system.tables with JSONEachRow format
    const resultSet = await client.query({
      query: 'SELECT * FROM system.tables',
      format: 'JSONEachRow',
    });

    // Stream the results and collect them
    const results = [];
    
    for await (const row of resultSet.stream()) {
      results.push(row);
    }

    // Log the fully parsed JSON array to the console
    console.log(JSON.stringify(results, null, 2));
    
  } catch (error) {
    console.error('Error executing query:', error);
    process.exit(1);
  } finally {
    await client.close();
  }
}

// Execute the query
queryTables();